tyk-oss/values.yaml

```diff
@@ -1,4 +1,5 @@
 global:
+  isDataplane: false
```

Comment:
by common definition, the gateway and OSS chart is also a data plane. So I'm wondering what's the meaning of this field?

Reply:
In my opinion, the "data plane" is just a special  configuration of OSS/gateway to be able to do the "assigned" role in the documentation. This variable just verifies whether we are going for that configuration or not.